rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -144,7 +178,7 @@ def get_balances(
         nft_results = self._filter_ignored_nfts(_nft_results)
         cached_db_result = self.get_nfts_with_price()
         cached_db_prices = {x['asset']: x for x in cached_db_result}
-        db_data: List[Tuple[str, Optional[str], Optional[str], Optional[str], int, ChecksumEvmAddress]] = []  # noqa: E501
+        db_data: List[Tuple[str, Optional[str], Optional[str], Optional[str], int, ChecksumEvmAddress, bool, Optional[str], str, Optional[str]]] = []  # noqa: E501
```

Comment:
Can you move this type to a constant at the beginning of the file and mark there in the type whatrepresents each element in a comment? Something in the line of 

```
            entry[0],  # identifier
            entry[3],  # symbol
            entry[6],  # coingecko
            entry[7],  # cryptocompare
            entry[8],  # forked
            entry[4],  # started
```

(this is from another place not related to this)

**OR** maybe it would be even better to use a `NamedTuple`. This is the kind of things I would break mixing two column :sweat:

Reply:
Yeah, I'll change it somehow, good point :+1: